### DISCUSSION TITLE
Unable to easily tell when something worked.

### DISCUSSION THREAD

**Discussion by kpurintun** [TD22057/insteon-mqtt#427] [General]
I have recently entered the HA realm and starting to add my stuff in and work my way out of an indigo installation.

I am not sure if I was stupid at any point along the way.  The first thing I did was add the modem and all the device addresses and names into the config.yaml under the categories.  i then did the join-all, pair-all, and refresh-all.  I think this might have been a mistake.  i have 41 devices.  two of which are battery powered remotes...  I guess when these sleep, it goofs things up...  maybe they should be further down the list..  or 'sleepable' devices should be at the bottom.  anyways..  I think the '-all' commands seem to load the insteon network with chatter.  

So I started to 'join' then 'pair', then 'refresh' each.  repeating the step until it was successful.  many times it would fail on the first or second try.  

I also wouldn't see everything in an MQTT explorer until I clicked the switch several times...  

I guess my question is...  what can I do to see that each of my devices has 'fully loaded and is joined, paired, and working as intended'?  I am not running this in docker, but installed directly from HACS as a normal integration.

**Comment by krkeegan** [comment]
> I am not running this in docker, but installed directly from HACS as a normal integration.

Sadly, HomeAssistant installation options are so numerous and poorly named that I am not sure exactly what your installation method looks like.

I think the first detail that would be helpful to know is:

- How are you sending these commands?  From a terminal window? Via MQTT? Or do you have the WebGUI option?

The answer to your main question, `how can I tell that a command worked and completed` should be straight forward.  Each of the commands you have identified will output a series of messages including one when complete.
